### PR TITLE
try to add license and notice to jars

### PR DESCRIPTION
**Pull request by pjfanning**
relates to https://github.com/apache/incubator-pekko/issues/74

### CODE REVIEW

File: NOTICE [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+This product includes software developed at
+The Apache Software Foundation (https://www.apache.org/).
+
+This product contains significant parts that were originally based on software from Lightbend (Akka <https://akka.io/>).
+Copyright (C) 2009-2022 Lightbend Inc. <https://www.lightbend.com>
+
+Apache Pekko is a derived from Akka 2.6.x, the last version that was distributed under the
+Apache License, Version 2.0 License.
+
+`pekko-protobuf` includes protobuf v2.5.0 code developed by Google Inc.
```
Comment (justinmclean) [inline NOTICE:13]: There is no need for the pekko-protobuf line

Comment (pjfanning) [reply NOTICE:13]: @justinmclean I was basing this on other ASF project NOTICE files that often mention code from other projects that has been included in that project. Examples:
* https://github.com/apache/spark/blob/master/NOTICE
* https://github.com/apache/poi/blob/trunk/legal/NOTICE
* https://github.com/apache/xmlbeans/blob/trunk/NOTICE.txt

For Drill, the NOTICE says the other licenses are in the main LICENSE file
* https://github.com/apache/drill/blob/master/LICENSE

So I guess, there are differences in approach. Could you suggest if we need to acknowledge the Google copyright at all? We are planning to keep the code headers that already acknowledge this.

Comment (justinmclean) [reply NOTICE:13]: They should only mention that when the original code copyright statement has been removed. You only add the copyright statement to the NOTICE file when it's been relocated in this way.

Comment (justinmclean) [reply NOTICE]: Another instance where you add content to NOTICE is when you bundle some software that is Apache licensed and that software has a NOICE file. In that case, you need to look at the bundled software NOTICE file and copy what is required from it your NOTICE file.

Comment (justinmclean) [reply NOTICE]: So in the case of Google 1. Does the software have a NOTICE file? If so (and Google software generally doesn't) then you need to add it content to NOTICE 2. Has tech copyright header been removed (with permission)? If so then yes you need to add the copyright to NOTICE, this however seems unlikely. It's very likely that Google doesn't need to mentioned in the NOTICE file.

Comment (pjfanning) [reply NOTICE]: thanks @justinmclean - I have already removed the Google protobuf lines from the NOTICE. I didn't find anything in the protobuf 2.5.0 that requires that we update our NOTICE file.

https://github.com/protocolbuffers/protobuf/releases?q=v2.5.0&expanded=true

File: NOTICE [hunk 2]
```diff
@@ -0,0 +1,11 @@
+Apache Pekko
+Copyright 2022, 2023 The Apache Software Foundation
+
+This product includes software developed at
+The Apache Software Foundation (https://www.apache.org/).
```
Comment (seglo) [inline NOTICE:5]: Is it worth mentioning https://pekko.apache.org/ ?

**Review by seglo — APPROVED** [review]

File: NOTICE (hunk 2 above)
Comment (pjfanning) [reply NOTICE:5]: NOTICE files are generally added for ASF projects. I've never seen them include project specific URLs. An example is the Apache Spark NOTICE file.

https://github.com/apache/spark/blob/master/NOTICE

**Pull request merged**